src/shared/components/active-date/ActiveDate.tsx

```
@@ -0,0 +1,125 @@
+interface ActiveDateProps {
+  formAvailableDates: AvailableDate[];
+  availableDatesError?: string;
+  errorMessages:
+    | typeof FOOD_TRUCK_ERROR_MESSAGE
+    | typeof ESTIMATE_ERROR_MESSAGE;
+  maxLength: number;
+  handleActiveDateSetValue: (_value: AvailableDate[]) => void;
+  handleActiveDateError: (_message: string) => void;
+}
+
+export default function AvailableDate({
```

Biome (2.1.2)

[error] Shouldn't redeclare 'AvailableDate'. Consider to delete it or rename it.

'AvailableDate' is defined here:


(lint/suspicious/noRedeclare)